client/homebrew/navbar/navbar.less

```diff
@@ -1,272 +1,350 @@
+							transform : translateX(-50%) translateY(100%);
+						}
+						&.inactive {
+							opacity   : 0;
+							transform : translateX(-50%) translateY(0%);
 						}
+						.row {
+							display   : flex;
+							flex-flow : row wrap;
+							width     : 100%;
+							h4 {
+								box-sizing : border-box;
+								display    : block;
+								flex-basis : 20%;
+								flex-grow  : 1;
+								min-width  : 76px;
+								padding    : 5px 0;
+								color      : #BBBBBB;
+								text-align : center;
+							}
+							p {
+								flex-basis     : 80%;
+								flex-grow      : 1;
+								padding        : 5px 0;
+								font-family    : 'Open Sans', sans-serif;
+								font-size      : 10px;
+								font-weight    : normal;
+								text-transform : initial;
+								.tag {
+									display          : inline-block;
+									padding          : 2px;
+									margin           : 2px 2px;
+									background-color : #444444;
+									border           : 2px solid grey;
+									border-radius    : 5px;
+								}
+								a.userPageLink {
+									color           : white;
+									text-decoration : none;
+									&:hover { text-decoration : underline; }
+								}
+							}
+							&:nth-of-type(even) { background-color : #555555; }
+						}
+					}
+				}
+				&.warning {
+					position         : relative;
+					color            : white;
+					background-color : @orange;
+					&:hover > .dropdown { visibility : visible; }
+					.dropdown {
+						position         : absolute;
+						top              : 28px;
+						left             : 0;
+						z-index          : 10000;
+						box-sizing       : border-box;
+						display          : block;
+						width            : 100%;
+						padding          : 13px 5px;
+						text-align       : center;
+						visibility       : hidden;
+						background-color : #333333;
 					}
 				}
-				&:nth-of-type(even) {
-					background-color : #555;
+				&.account {
+					min-width : 100px;
+					&.username { text-transform : none;}
+				}
+			}
+			.navDropdownContainer {
+				position : relative;
+				.navDropdown {
+					position   : absolute;
+					top        : 28px;
+					left       : 0px;
+					z-index    : 10000;
+					width      : 100%;
+					max-height : calc(100vh - 28px);
+					overflow   : hidden auto;
+					.navItem {
+						position           : relative;
+						display            : block;
+						width              : 100%;
+						padding            : 8px 5px;
+						border             : 1px solid #888888;
+						border-bottom      : 0;
+						animation-name     : glideDropDown;
+						animation-duration : 0.4s;
+					}
+				}
+				&.recent {
+					position : relative;
+					.navDropdown .navItem {
+						#backgroundColorsHover;
+						.animate(background-color);
+						position         : relative;
+						box-sizing       : border-box;
+						display          : block;
+						max-height       : ~"calc(100vh - 28px)";  // I don't think is correct syntax, but leaving it in for now... (Gazook89)
```

Comment:
if you are worried about the calc itself, it is valid, substracting 28px from the total height of the window. https://developer.mozilla.org/en-US/docs/Web/CSS/calc

the `~""` is invalidating the statement, so i don't get it either.

Reply:
Gitter today:
> From memory, that's to maintain the calc() as CSS, because otherwise LESS will try to execute it's own calc function
A quick Google shows that there is an option from v1.4 onwards that will disable this behaviour, but changes the required format for all actual LESS calculations

So we can leave that right there